bin/bovine.py

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env python3
+import json
+import os
+##########################
+# Read the Static Directory
+##########################
+
```

Comment:
We should decide right away if we want to do this in a functional style or object oriented style... Then move this code to a method or function.